pages/checkin.tsx

```diff
@@ -44,11 +44,19 @@ export default function CheckinPage() {
   const prefillPhone = router.query.phone?.toString()
   const prefillAddress = router.query.address?.toString()
 
-  const prefilledGuest = {
-    name: prefillName || guestInfo.data?.name,
-    phone: prefillPhone || guestInfo.data?.phone,
-    address: prefillAddress || guestInfo.data?.address,
-  }
+  const prefilledGuest =
+    prefillName ||
+    prefillPhone ||
+    prefillAddress ||
+    guestInfo.data?.name ||
+    guestInfo.data?.phone ||
+    guestInfo.data?.address
+      ? {
+          name: prefillName || guestInfo.data?.name,
+          phone: prefillPhone || guestInfo.data?.phone,
+          address: prefillAddress || guestInfo.data?.address,
+        }
+      : undefined
```

Comment:
fyi, this was causing the rememberMe checkbox to be always checked. This is a bug introduced by https://github.com/railslove/rcvr-app/pull/34. These changes fix this.